integrationtests/security-manager-it/pom.xml

```diff
@@ -57,9 +57,7 @@
                      <value>${testNGListener}</value>
                   </property>
                </properties>
-               
-               <argLine>-Xmx1024m -XX:MaxPermSize=256m
-                  -Dsun.nio.ch.bugLevel</argLine>
```

Comment:
Most of the integration tests run in the container anyway, so this should be irrelevant.